plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTest.java

```diff
@@ -41,7 +48,10 @@
 
   @Mock Godot godotMock;
   @Mock AssetPackManager assetPackManagerMock;
+  @Mock Task<Void> removePackSuccessTaskMock;
```

Comment:
You don't have to do this in this PR, but you may want to change these to more generic names if possible like:
````
@Mock Task<Void> voidSuccessTaskMock;
@Mock Task<Void> voidFailureTaskMock;
````

Then you can initialize them (i.e. call doAnswer to set up their onSuccess / onFailure) in a @Before setup method.

Could be helpful if you find yourself using them a lot, or defining a task per PlayAssetDelivery method.

Reply:
I just checked the codebase, since Task<AssetPackStates> will be reused, I think the naming with type approach is definitely better.